src/stylesheets/custom.scss

```diff
@@ -287,3 +297,12 @@
 .usa-table th {
   font-weight: 400;
 }
+
+// Revert original USWDS button styles to inline-block from new inline-flex
+.usa-button {
+  display: inline-block;
+}
```

Comment:
Just a thought on the button styling - I actually kind of like the new `display: inline-flex` property on `usa-button`. It prevents us from having to use either the `display-flex` or margin utility classes on buttons to center the content.

If the only reason we're reverting to `inline-block` is to fix the table header sort icons, removing the `position-absolute` util class on the sort icons [here](https://github.com/CMS-Enterprise/easi-app/blob/6fc154d47380c564f670827196fb9de6d7f9e782/src/utils/tableSort.tsx#L25) achieves the same result.

I'm approving as-is, but just wanted to bring it up as another option!